docs/api-reference/core/timeline.md

```diff
@@ -89,6 +92,15 @@ Prevent timeline time from being updated by calls to `update`.
 
 Reset timeline time to `0`.
 
+### attachAnimation(animation: Object, [channelHandle : Number]) : Number
```

Comment:
For some reason I thought we were going to merge `addChannel`/`removeChannel` and `attachAnimation`/`detachAnimation` into a single pair of functions. But maybe that is not practical?

Even if we don't, is there any reason not to standardize on `add`/`remove` verbs (instead of `attach`/`detach`.

Reply:
The channel/animation thing was discussed in the RFC PR: https://github.com/uber/luma.gl/pull/1100#issuecomment-492439085

I want to distinguish between things that are part of the system (channel is a component of timeline) vs things that are simply linked to it (animations are independent systems that can be "hooked up" to a timeline but aren't part of it). I went with add/remove vs attach/detach to reflect this difference, but I'm open to suggestions for the naming.

Reply:
> The channel/animation thing was discussed in the RFC PR

Thanks, yes looks like I misunderstood that comment, my fault for not reading carefully enough, and fine either way.

But even with two categories of objects I think the fact that they have different names/nouns (`channels`/`animations) and different pairs of methods are enough to emphasize that they are two categories. 

Using `add`/`remove` for both should be fine. To me, using different verbs seems to suggest some subtle difference in semantics (e.g. around how the app should manage ownership of these objects) that doesn't seem to be the case.